tests/test_coredump.py

```diff
@@ -228,3 +232,9 @@ def test_coredump(helpers: conftest.Helpers) -> None:
     image = rootfs_image(helpers.root().joinpath("../nix/not-os-image.nix"))
     with spawn_qemu(image) as vm:
         run_vmsh_command(["coredump", str(vm.pid)])
+
+
+def test_ioctl_injection(helpers: conftest.Helpers) -> None:
```

Comment:
I guess I move the VM stuff into the helper code so we can separate tests later better.